js/index.ts

```diff
@@ -834,6 +837,8 @@ export class SharedPromiseCache {
   }
 }
 
+// if source is a string, create a browser FetchSource().
```

Comment:
This should be using /** type of comments to show in the docs as far as I know.

Reply:
Might be worth adding this on the conductor method too, this way you would see it in vscode when you write the code to use the constructor.